2013/07/12 18:38:59 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'fahrenheit.profile_picture' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `profile_picture`.
in /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/protected/views/user/index.php (510)
in /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/protected/controllers/UserController.php (234)
in /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/index.php (13)
2013/07/12 18:38:59 [error] [exception.CDbException] exception 'CDbException' with message 'The table "profile_picture" for active record class "ProfilePicture" cannot be found in the database.' in /Applications/XAMPP/xamppfiles/htdocs/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(ProfilePicture))
#1 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/db/ar/CActiveRecord.php(402): CActiveRecord::model('ProfilePicture')
#2 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/db/ar/CActiveRecord.php(77): CActiveRecord->getMetaData()
#3 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/protected/views/user/index.php(510): CActiveRecord->__construct()
#4 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CBaseController.php(126): require('/Applications/X...')
#5 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/Applications/X...', Array, true)
#6 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(869): CBaseController->renderFile('/Applications/X...', Array, true)
#7 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#8 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/protected/controllers/UserController.php(234): CController->render('index', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#10 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#13 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(291): CFilterChain->run()
#17 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('')
#19 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#20 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/fahrenheit/
HTTP_REFERER=http://localhost/fahrenheit/site/login
---
2013/07/12 18:45:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php:179
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(RWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('cad')
#8 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/cad')
#9 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fahrenheit/user/cad
---
2013/07/12 18:46:29 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php:179
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(RWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('cad')
#8 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/cad')
#9 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fahrenheit/user/cad
---
2013/07/12 18:51:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php:179
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/auth/CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(RWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('cad')
#8 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/cad')
#9 /Applications/XAMPP/xamppfiles/htdocs/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/fahrenheit/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fahrenheit/user/cad
---
2013/07/13 01:36:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/files/...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:36:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/files/t1436225470.png".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/files/...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/files/t1436225470.png
HTTP_REFERER=http://localhost/
---
2013/07/13 01:36:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/files/...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:36:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/files/t1436225470.png".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/files/...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/files/t1436225470.png
HTTP_REFERER=http://localhost/
---
2013/07/13 01:36:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/files/t1436225470.png".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/files/...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/files/t1436225470.png
HTTP_REFERER=http://localhost/
---
2013/07/13 01:37:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:37:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:38:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/status/create".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/status...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/status/create
HTTP_REFERER=http://localhost/
---
2013/07/13 01:38:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "teamsoft/status/create".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('teamsoft/status...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/teamsoft/status/create
HTTP_REFERER=http://localhost/
---
2013/07/13 01:41:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:41:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:41:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:41:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:45:32 [error] [exception.CException] exception 'CException' with message 'Property "CHttpRequest.basePath" is not defined.' in /opt/lampp/yii/framework/base/CComponent.php:130
Stack trace:
#0 /opt/lampp/testidocs/protected/views/status/_index.php(12): CComponent->__get('basePath')
#1 /opt/lampp/yii/framework/web/CBaseController.php(126): require('/opt/lampp/test...')
#2 /opt/lampp/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/opt/lampp/test...', Array, true)
#3 /opt/lampp/yii/framework/web/CController.php(869): CBaseController->renderFile('/opt/lampp/test...', Array, true)
#4 /opt/lampp/testidocs/protected/controllers/StatusController.php(75): CController->renderPartial('_index', Array)
#5 /opt/lampp/yii/framework/web/actions/CInlineAction.php(49): StatusController->actionCreate()
#6 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /opt/lampp/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('create')
#15 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('status/create')
#16 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /opt/lampp/testidocs/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/status/create
HTTP_REFERER=http://localhost/
---
2013/07/13 01:46:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 01:47:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in /opt/lampp/yii/framework/web/CController.php:483
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(270): CController->missingAction('index')
#1 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('index')
#2 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/index')
#3 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /opt/lampp/testidocs/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/user/index
HTTP_REFERER=http://localhost/
---
2013/07/13 01:48:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "loading.gif".' in /opt/lampp/yii/framework/web/CController.php:483
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(270): CController->missingAction('loading.gif')
#1 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('loading.gif')
#2 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/loading.gi...')
#3 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /opt/lampp/testidocs/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/user/loading.gif
HTTP_REFERER=http://localhost/user/dashboard
---
2013/07/13 01:48:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/user/dashboard
---
2013/07/13 01:48:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "loading.gif".' in /opt/lampp/yii/framework/web/CController.php:483
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(270): CController->missingAction('loading.gif')
#1 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('loading.gif')
#2 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/loading.gi...')
#3 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /opt/lampp/testidocs/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/user/loading.gif
HTTP_REFERER=http://localhost/user/dashboard
---
2013/07/13 01:49:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "loading.gif".' in /opt/lampp/yii/framework/web/CController.php:483
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(270): CController->missingAction('loading.gif')
#1 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('loading.gif')
#2 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/loading.gi...')
#3 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /opt/lampp/testidocs/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/user/loading.gif
HTTP_REFERER=http://localhost/user/dashboard
---
2013/07/13 01:50:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "loading.gif".' in /opt/lampp/yii/framework/web/CController.php:483
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(270): CController->missingAction('loading.gif')
#1 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('loading.gif')
#2 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/loading.gi...')
#3 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /opt/lampp/testidocs/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/user/loading.gif
HTTP_REFERER=http://localhost/user/dashboard
---
<<<<<<< Updated upstream
2013/07/13 02:36:19 [error] [exception.CException] exception 'CException' with message 'Property "EJcrop.success" is not defined.' in /opt/lampp/yii/framework/base/CComponent.php:173
Stack trace:
#0 /opt/lampp/yii/framework/web/CWidgetFactory.php(161): CComponent->__set('success', 'js:alert("wow")')
#1 /opt/lampp/yii/framework/web/CBaseController.php(146): CWidgetFactory->createWidget(Object(ProfilePictureController), 'ext.jcrop.EJcro...', Array)
#2 /opt/lampp/yii/framework/web/CBaseController.php(172): CBaseController->createWidget('ext.jcrop.EJcro...', Array)
#3 /opt/lampp/testidocs/protected/views/profilePicture/crop.php(39): CBaseController->widget('ext.jcrop.EJcro...', Array)
#4 /opt/lampp/yii/framework/web/CBaseController.php(126): require('/opt/lampp/test...')
#5 /opt/lampp/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/opt/lampp/test...', Array, true)
#6 /opt/lampp/yii/framework/web/CController.php(869): CBaseController->renderFile('/opt/lampp/test...', Array, true)
#7 /opt/lampp/yii/framework/web/CController.php(782): CController->renderPartial('crop', Array, true)
#8 /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php(242): CController->render('crop', Array)
#9 /opt/lampp/yii/framework/web/actions/CInlineAction.php(49): ProfilePictureController->actionCrop()
#10 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#13 /opt/lampp/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#17 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('crop')
#19 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('profilePicture/...')
#20 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /opt/lampp/testidocs/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/profilePicture/crop
HTTP_REFERER=http://localhost/
---
2013/07/13 10:34:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xampp".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('xampp')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/xampp/
=======
2013/07/13 04:30:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /home/admin/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/admin/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/admin/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/admin/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/admin/yii/framework/base/CApplication.php(438): CModule->getComponent('db')
#4 /home/admin/yii/framework/db/ar/CActiveRecord.php(623): CApplication->getDb()
#5 /home/admin/yii/framework/db/ar/CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 /home/admin/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(ProfilePicture))
#7 /home/admin/yii/framework/db/ar/CActiveRecord.php(402): CActiveRecord::model('ProfilePicture')
#8 /home/admin/yii/framework/db/ar/CActiveRecord.php(77): CActiveRecord->getMetaData()
#9 /home/admin/public_html/protected/controllers/ProfilePictureController.php(242): CActiveRecord->__construct()
#10 /home/admin/yii/framework/web/actions/CInlineAction.php(49): ProfilePictureController->actionCrop()
#11 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#12 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#14 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#18 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('crop')
#20 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('profilePicture/...')
#21 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#22 /home/admin/public_html/index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/profilePicture/crop
HTTP_REFERER=http://www.fahrenheitracing.com/
---
2013/07/13 04:32:06 [error] [exception.CHttpException.404] exception 'CHttpException' in /home/admin/public_html/protected/extensions/yiinfinite-scroll/YiinfiniteScroller.php:43
Stack trace:
#0 /home/admin/yii/framework/web/CBaseController.php(173): YiinfiniteScroller->run()
#1 /home/admin/public_html/protected/views/user/index.php(274): CBaseController->widget('ext.yiinfinite-...', Array)
#2 /home/admin/yii/framework/web/CBaseController.php(126): require('/home/admin/pub...')
#3 /home/admin/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/home/admin/pub...', Array, true)
#4 /home/admin/yii/framework/web/CController.php(869): CBaseController->renderFile('/home/admin/pub...', Array, true)
#5 /home/admin/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#6 /home/admin/public_html/protected/controllers/UserController.php(243): CController->render('index', Array)
#7 /home/admin/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#8 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#15 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('dashboard')
#17 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/dashboard')
#18 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/admin/public_html/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user/dashboard?page=2
HTTP_REFERER=http://www.fahrenheitracing.com/
---
2013/07/13 04:34:46 [error] [exception.CHttpException.404] exception 'CHttpException' in /home/admin/public_html/protected/extensions/yiinfinite-scroll/YiinfiniteScroller.php:43
Stack trace:
#0 /home/admin/yii/framework/web/CBaseController.php(173): YiinfiniteScroller->run()
#1 /home/admin/public_html/protected/views/user/index.php(274): CBaseController->widget('ext.yiinfinite-...', Array)
#2 /home/admin/yii/framework/web/CBaseController.php(126): require('/home/admin/pub...')
#3 /home/admin/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/home/admin/pub...', Array, true)
#4 /home/admin/yii/framework/web/CController.php(869): CBaseController->renderFile('/home/admin/pub...', Array, true)
#5 /home/admin/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#6 /home/admin/public_html/protected/controllers/UserController.php(243): CController->render('index', Array)
#7 /home/admin/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#8 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#15 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('dashboard')
#17 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/dashboard')
#18 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/admin/public_html/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user/dashboard?page=2
HTTP_REFERER=http://fahrenheitracing.com/
---
2013/07/13 04:46:15 [error] [exception.CHttpException.404] exception 'CHttpException' in /home/admin/public_html/protected/extensions/yiinfinite-scroll/YiinfiniteScroller.php:43
Stack trace:
#0 /home/admin/yii/framework/web/CBaseController.php(173): YiinfiniteScroller->run()
#1 /home/admin/public_html/protected/views/user/index.php(274): CBaseController->widget('ext.yiinfinite-...', Array)
#2 /home/admin/yii/framework/web/CBaseController.php(126): require('/home/admin/pub...')
#3 /home/admin/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/home/admin/pub...', Array, true)
#4 /home/admin/yii/framework/web/CController.php(869): CBaseController->renderFile('/home/admin/pub...', Array, true)
#5 /home/admin/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#6 /home/admin/public_html/protected/controllers/UserController.php(243): CController->render('index', Array)
#7 /home/admin/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#8 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#15 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('dashboard')
#17 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/dashboard')
#18 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/admin/public_html/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user/dashboard?page=2
HTTP_REFERER=http://fahrenheitracing.com/
---
2013/07/13 04:49:09 [error] [exception.CHttpException.404] exception 'CHttpException' in /home/admin/public_html/protected/extensions/yiinfinite-scroll/YiinfiniteScroller.php:43
Stack trace:
#0 /home/admin/yii/framework/web/CBaseController.php(173): YiinfiniteScroller->run()
#1 /home/admin/public_html/protected/views/user/index.php(274): CBaseController->widget('ext.yiinfinite-...', Array)
#2 /home/admin/yii/framework/web/CBaseController.php(126): require('/home/admin/pub...')
#3 /home/admin/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/home/admin/pub...', Array, true)
#4 /home/admin/yii/framework/web/CController.php(869): CBaseController->renderFile('/home/admin/pub...', Array, true)
#5 /home/admin/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#6 /home/admin/public_html/protected/controllers/UserController.php(243): CController->render('index', Array)
#7 /home/admin/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#8 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#15 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('dashboard')
#17 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/dashboard')
#18 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/admin/public_html/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user/dashboard?page=2
HTTP_REFERER=http://fahrenheitracing.com/
---
2013/07/13 04:49:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "sankalp".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('sankalp')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sankalp
---
2013/07/13 11:08:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 11:10:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 12:27:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 12:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 12:32:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 12:37:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/admin/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/admin/public_html/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/07/13 15:20:10 [error] [exception.CHttpException.404] exception 'CHttpException' in /home/admin/public_html/protected/extensions/yiinfinite-scroll/YiinfiniteScroller.php:43
Stack trace:
#0 /home/admin/yii/framework/web/CBaseController.php(173): YiinfiniteScroller->run()
#1 /home/admin/public_html/protected/views/user/index.php(274): CBaseController->widget('ext.yiinfinite-...', Array)
#2 /home/admin/yii/framework/web/CBaseController.php(126): require('/home/admin/pub...')
#3 /home/admin/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/home/admin/pub...', Array, true)
#4 /home/admin/yii/framework/web/CController.php(869): CBaseController->renderFile('/home/admin/pub...', Array, true)
#5 /home/admin/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#6 /home/admin/public_html/protected/controllers/UserController.php(243): CController->render('index', Array)
#7 /home/admin/yii/framework/web/actions/CInlineAction.php(49): UserController->actionDashboard()
#8 /home/admin/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/admin/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/admin/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/admin/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/admin/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/admin/yii/framework/web/CController.php(291): CFilterChain->run()
#15 /home/admin/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/admin/yii/framework/web/CWebApplication.php(282): CController->run('dashboard')
#17 /home/admin/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/dashboard')
#18 /home/admin/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/admin/public_html/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user/dashboard?page=3
HTTP_REFERER=http://fahrenheitracing.com/
>>>>>>> Stashed changes
---
2013/07/13 14:31:52 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /opt/lampp/yii/framework/web/CController.php:336
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('create')
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('todo/create')
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/todo/create
HTTP_REFERER=http://localhost/todo/
---
2013/07/13 14:48:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
---
2013/07/13 14:48:28 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/
---
2013/07/13 14:48:29 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/
---
2013/07/13 14:48:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/
---
2013/07/13 15:16:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('create')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/user/create
HTTP_REFERER=http://localhost/site/login
---
2013/07/13 15:16:10 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/user/create
---
2013/07/13 15:18:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/site/login
---
2013/07/13 15:18:25 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'There must be at least one superuser!' in /opt/lampp/testidocs/protected/modules/rights/components/RAuthorizer.php:305
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/controllers/AssignmentController.php(51): RAuthorizer->getSuperusers()
#1 /opt/lampp/yii/framework/web/CController.php(1144): AssignmentController->accessRules()
#2 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#5 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('rights')
#8 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /opt/lampp/testidocs/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/rights
---
2013/07/13 15:19:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'There must be at least one superuser!' in /opt/lampp/testidocs/protected/modules/rights/components/RAuthorizer.php:305
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/controllers/AssignmentController.php(51): RAuthorizer->getSuperusers()
#1 /opt/lampp/yii/framework/web/CController.php(1144): AssignmentController->accessRules()
#2 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#5 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('user')
#7 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('rights/assignme...')
#8 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /opt/lampp/testidocs/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/rights/assignment/user/id/10
HTTP_REFERER=http://localhost/rights/assignment/user/id/10
---
2013/07/13 15:20:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 15:20:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/admin
---
2013/07/13 15:21:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/admin
---
2013/07/13 15:21:30 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/admin
---
2013/07/13 15:22:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
---
2013/07/13 15:22:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
HTTP_REFERER=http://localhost/
---
2013/07/13 15:22:43 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /opt/lampp/testidocs/protected/modules/rights/components/RController.php:62
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/components/RightsFilter.php(57): RController->accessDenied()
#1 /opt/lampp/yii/framework/web/filters/CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): CFilter->filter(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /opt/lampp/testidocs/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/user/admin
---
2013/07/13 15:22:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'There must be at least one superuser!' in /opt/lampp/testidocs/protected/modules/rights/components/RAuthorizer.php:305
Stack trace:
#0 /opt/lampp/testidocs/protected/modules/rights/controllers/AssignmentController.php(51): RAuthorizer->getSuperusers()
#1 /opt/lampp/yii/framework/web/CController.php(1144): AssignmentController->accessRules()
#2 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#3 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#4 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#5 /opt/lampp/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /opt/lampp/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('rights')
#8 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /opt/lampp/testidocs/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/rights
---
2013/07/13 19:09:07 [error] [php] require(../../extensions/jcrop/Resize.php): failed to open stream: No such file or directory (/opt/lampp/testidocs/protected/controllers/ProfilePictureController.php:209)
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): ProfilePictureController->runAction()
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): RightsFilter->filter()
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): ProfilePictureController->filterRights()
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): ProfilePictureController->runActionWithFilters()
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): ProfilePictureController->run()
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CWebApplication->run()
REQUEST_URI=/profilePicture/upload
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/modules/rights/components/RController.php (36)
2013/07/13 19:09:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t1876393140.png".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t18763931...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t1876393140.png
HTTP_REFERER=http://localhost/
---
2013/07/13 19:09:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t1876393140.png".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t18763931...')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t1876393140.png
HTTP_REFERER=http://localhost/
---
2013/07/13 19:09:19 [error] [php] require(../../extensions/jcrop/Resize.php): failed to open stream: No such file or directory (/opt/lampp/testidocs/protected/controllers/ProfilePictureController.php:209)
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): ProfilePictureController->runAction()
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): RightsFilter->filter()
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): ProfilePictureController->filterRights()
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): ProfilePictureController->runActionWithFilters()
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): ProfilePictureController->run()
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CWebApplication->run()
REQUEST_URI=/profilePicture/upload
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/modules/rights/components/RController.php (36)
2013/07/13 19:09:43 [error] [php] require(/../../extensions/jcrop/Resize.php): failed to open stream: No such file or directory (/opt/lampp/testidocs/protected/controllers/ProfilePictureController.php:209)
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): ProfilePictureController->runAction()
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): RightsFilter->filter()
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): ProfilePictureController->filterRights()
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): ProfilePictureController->runActionWithFilters()
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): ProfilePictureController->run()
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CWebApplication->run()
REQUEST_URI=/profilePicture/upload
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/modules/rights/components/RController.php (36)
2013/07/13 19:09:55 [error] [php] require(/../extensions/jcrop/Resize.php): failed to open stream: No such file or directory (/opt/lampp/testidocs/protected/controllers/ProfilePictureController.php:209)
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): ProfilePictureController->runAction()
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): RightsFilter->filter()
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): ProfilePictureController->filterRights()
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): ProfilePictureController->runActionWithFilters()
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): ProfilePictureController->run()
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CWebApplication->run()
REQUEST_URI=/profilePicture/upload
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/modules/rights/components/RController.php (36)
2013/07/13 19:10:20 [error] [php] require(/protected/extensions/jcrop/Resize.php): failed to open stream: No such file or directory (/opt/lampp/testidocs/protected/controllers/ProfilePictureController.php:209)
Stack trace:
#0 /opt/lampp/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/yii/framework/web/filters/CFilterChain.php(133): ProfilePictureController->runAction()
#2 /opt/lampp/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /opt/lampp/testidocs/protected/modules/rights/components/RController.php(36): RightsFilter->filter()
#4 /opt/lampp/yii/framework/web/filters/CInlineFilter.php(58): ProfilePictureController->filterRights()
#5 /opt/lampp/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /opt/lampp/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /opt/lampp/yii/framework/web/CController.php(265): ProfilePictureController->runActionWithFilters()
#8 /opt/lampp/yii/framework/web/CWebApplication.php(282): ProfilePictureController->run()
#9 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /opt/lampp/testidocs/index.php(13): CWebApplication->run()
REQUEST_URI=/profilePicture/upload
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/controllers/ProfilePictureController.php (209)
in /opt/lampp/testidocs/protected/modules/rights/components/RController.php (36)
2013/07/13 19:10:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
2013/07/13 19:30:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/parry
---
2013/07/13 19:32:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/parry
---
2013/07/13 19:32:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/parry
---
2013/07/13 19:34:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/parry
---
2013/07/13 19:35:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "files/t2689577.jpg".' in /opt/lampp/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('files/t2689577....')
#1 /opt/lampp/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /opt/lampp/testidocs/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/files/t2689577.jpg
HTTP_REFERER=http://localhost/
---
